src/utils/helper_functions/generate_test_data.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
I didn't know this, thank you for spreading the knowledge! 
Interesting that you call it a `hash-bang` I usually call it a `shebang`

Reply:
@MikeLippincott - just a minor note here on the naming convention: I tend to prefer using the literal "hash == #" and "bang == !" version to avoid gender-specific connotations which could be inferred. Mostly the intent on my part here is towards doing no harm. I believe the origins of "shebang" might be obscure, so it's hard to say what the "she" part specifically came from or what it might imply.